addons/mail/static/src/web/suggested_recipient.xml

```diff
@@ -6,7 +6,7 @@
                 <input t-attf-id="{{ props.recipient.id }}_checkbox" class="form-check-input" type="checkbox" t-att-checked="props.recipient.checked" t-on-change="onChangeCheckbox"/>
                 <label class="form-check-label" t-attf-for="{{ props.recipient.id }}_checkbox">
                     <t t-if="props.recipient.name" t-esc="props.recipient.name"/>
-                    <t t-if="props.recipient.email">
+                    <t t-if="props.recipient.email and props.recipient.email != props.recipient.name">
```

Comment:
it is good practice to use [strict inequality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality) in JavaScript
```suggestion
                    <t t-if="props.recipient.email and props.recipient.email !== props.recipient.name">
```

Reply:
Mhhh, I see that in the code we mostly use `!=`, are you sure? new in saas-16.3+?
Also, `and` & `!==` on the same line, seems a bit confusing :sweat_smile:

Reply:
Most of the templates are rendered on the Python side, hence the under-representation of `!==`. But if you just look at the JS side, for example by restricting your search to the `static/src/` folder, you'll see that `!==` is actually the most widespread there

> Also, and & !== on the same line, seems a bit confusing 😅

This is because you can't put unescaped ampersands in an xml, so we just write `and` instead, and under the hood Owl replaces it with `&&`

Reply:
Thanks for info! Updated :+1: